web/app/themes/radcliffe/search.php

```diff
@@ -1,13 +1,12 @@
 <?php get_header(); ?>
 
-<div class="content">
-
-    <div class="section-inner">
-        <form method="get" class="search-results-page-form search-form"
-              action="<?php echo esc_url(home_url('/')); ?>">
-            <label class="search-term-input-label" for="s" >Search for: </label>
-            <input type="search" placeholder="<?php _e('enter search term', 'radcliffe'); ?>" name="s" id="s" class="search-term-input" />
-            <input type="submit" value="<?php _e('Search', 'radcliffe'); ?>" class="search-button">
+<main class="content" id="maincontent">
+
+    <div id="search2" class="section-inner">
+        <form class="search-form" method="get" action="<?php echo esc_url(home_url('/')); ?>">
+            <label class='search-term-input-label no-focus' for="s">Search this website for: </label>
+            <input type="search" name="s" id="s" class="search-term-input no-focus"/>
+            <input type="submit" value="Search" class="no-focus">
```

Comment:
This file has been improved for accessibility yet has been stript of locale functions and placeholder form attributes for other viewers. Can we try and keep focus on making our sites a pleasure for all that view it.

We need the locale functions and placeholder attributes and text returned.

Nb. if you write and static text in a PHP file, it must be wrapped in a WordPress locale function. Without this we are excluding the possibility of people that cannot read English from consuming the site if this becomes a requirement - foreign languages are also an accessibility consideration.

Reply:
The search instances have been problematic – too complex for me. Hoping you or Rob can sort them out. As above, the placeholder can be problematic – but the article proposes an alternative to make it the best of both worlds. Ya?